### DISCUSSION TITLE
Admin local login only

### DISCUSSION THREAD

**Discussion by luigir-it** [seerr-team/seerr#603] [General]
Hi. 
I have jellyseerr (with multiple users) exposed to the web.
I'd like to deny access to the root user when using the public link and allow only through the internal link (lan). 
This I think is a much safer approach. I can do this in jellyfin, I was wondering if it is possible in jellyseerr too.

**Comment by fallenbagel** [comment]
What do you mean by root user? Admin?

If you disable login on jellyfin for a specific user they wouldn't be able to login to jellyseerr using jellyfin

**Reply by luigir-it** [reply]
Yes sorry I meant admin. I [disabled login ONLY from outsite lan in jellyfin](https://jellyfin.org/docs/general/server/users/adding-managing-users#profile) (by unchecking `Allow remote connections to this Jellyfin Server`)  but I can still login in jellyseerr.

**Reply by fallenbagel** [reply]
> Yes sorry I meant admin. I [disabled login ONLY from outsite lan in jellyfin](https://jellyfin.org/docs/general/server/users/adding-managing-users#profile) (by unchecking `Allow remote connections to this Jellyfin Server`)  but I can still login in jellyseerr.
> 
> 

That's because the connection between jellyseerr and jellyfin is still local. This will be fixed once the forwardauth is added.

A simple workaround would be to change the connection between to use the reverse proxy or remote but not recommended.

A recommended would be ig to disable the admin user and create a local admin user and use jellyseerr login for that

**Reply by luigir-it** [reply]
> disable the admin user and create a local admin user and use jellyseerr login

Could  you expand on this? You're saying to create an admin account using jellyseerr login instead of using the admin user of jellyfin? Why would it be "safer"? I'll still be able to login from outsite the lan.

**Reply by fallenbagel** [reply]
> > disable the admin user and create a local admin user and use jellyseerr login
> 
> Could  you expand on this? You're saying to create an admin account using jellyseerr login instead of using the admin user of jellyfin? Why would it be "safer"? I'll still be able to login from outsite the lan.

Safer in the sense that if you go w jellyfin then they get access to both jellyfin and jellyseerr. But w jellyseerr only jellyseerr if you get hacked. Thats what I mean. Plus you can setup fail2ban to ban brute forces. 

You should be fairly safe tbh even if you don't as long as there isn't a person who's targeting you specifically.
